roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -57,6 +65,14 @@ def subclasses(cls):
             for subclass in subclass.subclasses():
                 yield subclass
 
+    @staticmethod
```

Comment:
@sosiouxme I've brought the code back to the module level, since it depends on module/file-level stuff like `__package__` and `__file__`. Please let me know what you think of the current state.